<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewAttribute</class>
 <widget class="QDialog" name="CreateNewAttribute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Attribute</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Categ">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Please select a category:</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="sft_radio">
       <property name="text">
        <string>Safety</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sct_radio">
       <property name="text">
        <string>Security</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="avb_radio">
       <property name="text">
        <string>Availability</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cmp_radio">
       <property name="text">
        <string>Completeness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ftl_radio">
       <property name="text">
        <string>Failure Tolerance</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="acc_categ">
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit_categ">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
